src/components/Chat/ChatFrame.js

```diff
@@ -13,13 +13,11 @@ import ReactMarkdown from "react-markdown";
 import {CloseOutlined} from "@material-ui/icons";
 import UserStatusDisplay from "../Lobby/UserStatusDisplay";
 import InfiniteScroll from 'react-infinite-scroller';
-import EmojiPickerPopover from "./EmojiPickerPopover";
-
-// const emojiSupport = text => text.value.replace(/:\w+:/gi, name => emojiIndex.search(name).map(o=>o.native));
+// import EmojiPickerPopover from "./EmojiPickerPopover";
+import styled from 'styled-components';
```

Comment:
When adding new packages, please *also* add them to package.json, so that it is easier for everyone else to understand the fact that this requires a new dependency, and what that dependency is.